modules/exploits/windows/http/exchange_ecp_dlp_policy.rb

```diff
@@ -3,6 +3,8 @@
 # Current source: https://github.com/rapid7/metasploit-framework
 ##
 
+require 'faker'
```

Comment:
Curious that this has to be `require`'d now. Something related to Zeitwerk?

Reply:
cc @adfoster-r7

Reply:
Thanks for calling this out, will look into this more and cross-reference this comment when we've got a PR to fix things up :+1:

Reply:
Cross-referencing long-term fix https://github.com/rapid7/metasploit-framework/pull/14816

Reply:
Thank you!